sdk/appcenter-crashes/src/test/java/com/microsoft/appcenter/crashes/WrapperSdkExceptionManagerTest.java

```diff
@@ -179,43 +182,49 @@ public void saveWrapperSdkCrash() throws JSONException, IOException {
     }
 
     @Test
+    @PrepareForTest({android.util.Log.class})
```

Comment:
Just curiosity, any reason of specifying this in the method?

Reply:
When you mock only 1 time and not in the entire test class you can do that.

Reply:
Remove the curly braces though for 1 element.